test/smoke/shared_memory_ttd_region/shared_memory_ttd_region.c

```diff
@@ -0,0 +1,32 @@
+#include <omp.h>
+#include <stdio.h>
+
+#define ITERS 4096
+
+int foo(int a) {
```

Comment:
is the argument necessary? appears unused

Reply:
unnecessary, will remove